### PR TITLE
Add script to replace `Prettier stable`/`Prettier master` with the actual versions

### PR DESCRIPTION
**Pull request by sosukesuzuki**
Context: https://github.com/prettier/prettier/issues/8882#issuecomment-671033909

Adds a script to replace `Prettier stable`/`Prettier master` => `Prettier 2.0`/`Prettier 2.1`.

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by fisker** [comment]
Why not put it here https://github.com/prettier/prettier/blob/7c318cc677c19e64ed7efb72376cd26f1f72d613/scripts/release/steps/update-changelog.js#L44, so we can replace it automatically on release

**Comment by fisker** [comment]
And we can just simply

``text.replace(/Prettier stable/g, `Prettier ${previousVersion}`)``
``text.replace(/Prettier master/g, `Prettier ${version}`)``

**Comment by sosukesuzuki** [comment]
> Why not put it here

I didn't know well about release script. Thank you.

File: scripts/release/steps/update-changelog.js [hunk 1]
```diff
@@ -30,6 +30,29 @@ function writeChangelog({ version, previousVersion, releaseNotes }) {
...
+  const blogPostData = fs.readFileSync(blogPost, "utf-8");
+  const newBlogPostData = blogPostData
+    .split("\n")
+    .map((line) => {
+      const trimmed = line.trim();
+      const isComments =
+        trimmed.startsWith("// ") ||
+        trimmed.startsWith("# ") ||
+        (trimmed.startsWith("<!-- ") && trimmed.endsWith(" -->")) ||
+        (trimmed.startsWith("/* ") && trimmed.endsWith(" /*")) ||
```
Comment (fisker) [inline scripts/release/steps/update-changelog.js:43]:
```suggestion
        (trimmed.startsWith("/* ") && trimmed.endsWith(" */")) ||
```

**Comment by fisker** [comment]
This should be able to test this script


```bash
node scripts/draft-blog-post.js
node -e "require('./scripts/release/steps/update-changelog')({version:'2.1.0', previousVersion: '2.1.0'})"
````

**Comment by sosukesuzuki** [comment]
> Do you think it's necessary to check it's in comments? Didn't see anyone use Prettier stable/master in other place.

The format of changelog text is not defined, so I thought it's necessary to check there is  `Prettier stable/master` in comments.
But I have no strong opinion.

File: scripts/release/steps/update-changelog.js [hunk 2]
```diff
@@ -30,6 +30,29 @@ function writeChangelog({ version, previousVersion, releaseNotes }) {
...
+        trimmed.startsWith("// ") ||
+        trimmed.startsWith("# ") ||
+        (trimmed.startsWith("<!-- ") && trimmed.endsWith(" -->")) ||
+        (trimmed.startsWith("/* ") && trimmed.endsWith(" */")) ||
+        (trimmed.startsWith("{{!-- ") && trimmed.endsWith(" --}}"));
+      if (!isComments) {
+        return line;
+      }
+      return line
+        .replace(/Prettier stable/g, `Prettier ${previousVersion}`)
```
Comment (fisker) [inline scripts/release/steps/update-changelog.js:49]: You use uppercase `Stable` in

https://github.com/prettier/prettier/blob/master/changelog_unreleased/javascript/pr-6319.md,

so maybe `/Prettier [Ss]table/g` or `/Prettier stable/ig`

**Comment by fisker** [comment]
I say, we don't need check that.

File: scripts/release/steps/update-changelog.js [hunk 3]
```diff
@@ -30,6 +30,22 @@ function writeChangelog({ version, previousVersion, releaseNotes }) {
...
+  const newBlogPostData = blogPostData
+    .split("\n")
+    .map((line) => {
+      return line
+        .replace(/prettier stable/gi, `Prettier ${formatVer(previousVersion)}`)
+        .replace(/prettier master/gi, `Prettier ${formatVer(version)}`);
+    })
+    .join("\n");
+  fs.writeFileSync(blogPost, newBlogPostData);
+}
```
Comment (fisker) [inline scripts/release/steps/update-changelog.js:47]:
```suggestion
function formatVer(ver) {
  return `${semver.major(ver)}.${semver.minor(ver)}`;
}

function replaceVersionInBlogPost({ blogPost, version, previousVersion }) {
  const blogPostData = fs.readFileSync(blogPost, "utf-8");
  const newBlogPostData = blogPostData .replace(/prettier stable/gi, `Prettier ${formatVer(previousVersion)}`)
        .replace(/prettier master/gi, `Prettier ${formatVer(version)}`);
  fs.writeFileSync(blogPost, newBlogPostData);
}
```

**Comment by fisker** [comment]
Please update this https://github.com/prettier/prettier/wiki/Release-Checklist , when merge

**Review by evilebottnawi — APPROVED** [review]

**Pull request merged**